cache/sampleStoreTimeout.js

```diff
@@ -27,10 +28,13 @@ const ONE = 1;
  * @param  {Array} samples - Samples to be timedout
  * @param  {Hash} aspects - Aspects hash, name-> object
  * @param  {Date} curr - Current datetime
- * @returns {Array} Commands needed to timeout samples
+ * @returns {Object} - with with sampCmds and timedOutSamples properties.
```

Comment:
duplicate with.
also timedOutSamples sound like samples that already timed out. Perhpas use samplesToTimeOut?